src/transaction/log_manager.c

```diff
@@ -13359,7 +13359,7 @@ cdc_make_timer_loginfo (time_t at_time, int trid, char *user, CDC_LOGINFO_ENTRY
 }
 
 static int
-cdc_find_user (THREAD_ENTRY * thread_p, LOG_PAGE * log_page, LOG_LSA process_lsa, int trid, char **user)
+cdc_find_user (THREAD_ENTRY * thread_p, LOG_LSA process_lsa, int trid, char **user)
```

Comment:
I hope that there is a function comment which describes its usage and restriction. It took me time to clarify what's this function for. I am still quite confused and guessing. Please clarify the parameters and the situation it is used.

Reply:
Why don't we need to know in the case of committed trxs?